docs/references/architecture/recipes/about-recipes.md

```diff
@@ -7,4 +7,6 @@ draft: true
 
 Tauri recipes are descriptions of use-cases that are entirely configurable within the `src-tauri/tauri.conf.json` file. These are not the limits of what Tauri can do, and there are probably more out there. If you discover one, why not get in touch and help us update this collection!
 
-If you haven't read about the general design of Tauri, then it would make the most sense for you to visit the ["Getting Started"](../../getting-started/beginning-tutorial) and become familiar with the basic architecture and terminology used in these recipes.
+If you haven't read about the general design of Tauri, then it would make the most sense for you to visit the ["Quick Start"][quick start] and become familiar with the basic architecture and terminology used in these recipes.
```

Comment:
```suggestion
If you haven't read about the general design of Tauri, then it would make the most sense for you to visit the [Quick Start] guide and become familiar with the basic architecture and terminology used in these recipes.
```